### PR TITLE
v0.7.0 release

### PR DESCRIPTION
**Pull request by scopsy**

### CODE REVIEW

File: providers/fcm/src/lib/fcm.provider.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      },
+      crypto.randomBytes(4).toString()
+    );
+    this.messaging = getMessaging(firebase);
+  }
+
+  async sendMessage(
+    options: IPushOptions
+  ): Promise<ISendMessageSuccessResponse> {
+    delete (options.overrides as any).notificationIdentifiers;
```
Comment (djabarovgeorge) [inline providers/fcm/src/lib/fcm.provider.ts:40]: Hey @scopsy @raikasdev 
what is the purpose of this one?
Looks like the test 'options.overrides' is undefined which causes the test to fails.

Comment (scopsy) [reply providers/fcm/src/lib/fcm.provider.ts:40]:
```suggestion
    delete (options.overrides as any)?.notificationIdentifiers;
```
It should not pass the identifiers from the overrides when sending a message. I think you can just do it like this.

Comment (djabarovgeorge) [reply providers/fcm/src/lib/fcm.provider.ts:40]: Awesome thanks, I think there is more i am looking into it rn

**Comment by scopsy** [comment]
@raikasdev I think the failing tests might be related to the changes you've introduced in the rollup config, remember why you changed it?

**Comment by raikasdev** [comment]
Rollup was throwing a error about missing type index, and the config was pointing to the wrong place, as the .d.ts was generated in the esm folder and not the esm/types folder

**Pull request merged**